chord_metadata_service/chord/ingest/phenopackets.py

```diff
@@ -252,24 +327,17 @@ def ingest_phenopacket(phenopacket_data: dict[str, Any],
     # Get or create all biosamples in the phenopacket
     biosamples_db = [get_or_create_biosample(bs) for bs in biosamples]
 
-    # Get or create all genes in the phenopacket
-    # TODO: May want to augment alternate_ids
-    genes_db = [get_or_create_gene(g) for g in genes]
-
-    # Get or create all diseases in the phenopacket
-    diseases_db = [get_or_create_disease(disease) for disease in diseases]
-
-    # Get or create all manually-specified HTS files in the phenopacket
-    hts_files_db = [get_or_create_hts_file(hts_file) for hts_file in hts_files]
-
     # Get or create all resources (ontologies, etc.) in the phenopacket
     resources_db = [ingest_resource(rs) for rs in resources]
 
+    interpretations_db = [get_or_create_interpretation(interp) for interp in interpretations]
+    diseases_db = [get_or_create_disease(disease) for disease in diseases]
+
     # Create phenopacket metadata object
     meta_data_obj = pm.MetaData(
         created_by=meta_data["created_by"],
         submitted_by=meta_data.get("submitted_by"),
-        phenopacket_schema_version="1.0.0-RC3",
+        phenopacket_schema_version="2.0.0",
```

Comment:
maybe if the version is specified in the phenopacket we should validate it against what we support here (2.0.0)

Reply:
Not sure I understand. The validation for metadata happens before this line, but I think it can be changed to this:
```python
phenopacket_schema_version=meta_data.get("phenopacket_schema_version")
```
Since the `phenopacket_schema_version` field is mandatory since 2.0.0 only.

Reply:
what I mean is that if the user-submitted meta data object has the `phenopacket_schema_version` field, we should validate that it's in a compatible range of katsu (i.e., is equal to 2.0.0 or if in the future we also support 2.0.X it is in `2.0.0-2.0.X`). I think us overwriting it to be 2.0.0 is fine, since it's how we actually represent phenopackets when they're served back to the user. not too important though.

Reply:
The phenopacket schema now validates against an array of supported versions, if validation passes we take the user-submitted value.